tests/integration/test_fim/test_registry/test_tags/test_registry_tags.py

```diff
@@ -6,38 +6,31 @@
 import pytest
 
 from wazuh_testing import global_parameters
-from wazuh_testing.fim import LOG_FILE_PATH, regular_file_cud, generate_params, delete_registry, registry_parser
+from wazuh_testing.fim import LOG_FILE_PATH, registry_value_cud, generate_params, delete_registry, registry_parser
```

Comment:
`delete_registry` and `registry_parser` are imported but unused.
```suggestion
from wazuh_testing.fim import LOG_FILE_PATH, registry_value_cud, generate_params
```